Algorithms/Sorters/BubbleSorter.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections.Generic;
+
+namespace Algorithms.Sorters
+{
+    public class BubbleSorter<T> : ISorter<T>
+    {       
+        public void Sort(T[] array, IComparer<T> comparer)
+        {
+            for (var i = 0; i < array.Length; i++)
```

Comment:
Think of case array.length = 1
```suggestion
            for (var i = 0; i < array.Length - 1; i++)
```